### PR TITLE
Attempt to fix IE grid

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Due to unusable rules in IE, reverted the navbar to use flexbox.

### CODE REVIEW

File: src/sections/ServicesHero/index.css [hunk 1]
```diff
@@ -5,7 +5,7 @@ $image-tablet-height: 340px;
 $image-mobile-height: $spacing-112;
 
 .root {
-  margin-top: $spacing-28;
```
Comment (taniaprocha) [inline src/sections/ServicesHero/index.css]: Since you are removing margin you can remove root class from js file

Comment (jfranciscosousa) [reply src/sections/ServicesHero/index.css]: Yes, I forgot to remove it during experiments.

File: src/components/Navbar/index.css [hunk 2]
```diff
@@ -8,23 +8,28 @@ $navbar-shadow: 0 2px 11px rgba(0, 0, 0, 0.1);
...
-
-  padding-top: $spacing-28;
+  padding: $spacing-12 0;
 
   a {
     text-decoration: none;
   }
 
   @include Breakpoint-mobileOnly {
+    display: grid;
```
Comment (taniaprocha) [inline src/components/Navbar/index.css]: You don't need grid on mobile navbar, right?

Comment (jfranciscosousa) [reply src/components/Navbar/index.css]: I do, for the menu itself.

Comment (taniaprocha) [reply src/components/Navbar/index.css]: So, we have also change to IE, right?

Comment (jfranciscosousa) [reply src/components/Navbar/index.css]: There is no menu in IE, unless the user changes the dimensions of the page

Comment (taniaprocha) [reply src/components/Navbar/index.css]: Forget 🤦‍♀️

**Pull request merged**